MediaRecorder.bs

```diff
@@ -223,8 +223,8 @@ interface MediaRecorder : EventTarget {
       </li>
 
       <li>If at any point, a track is added to or removed from the {{MediaRecorder/stream}}'s
-      track set, the UA MUST immediately stop gathering data, discard any data that it has gathered,
-      and queue a task, using the DOM manipulation task source, that runs the following steps:
+      track set, the UA MUST stop gathering data, and queue a task,
```

Comment:
What happened to "discard any data that it has gathered," ?

Reply:
See [this comment](https://github.com/w3c/mediacapture-record/issues/151#issuecomment-456756089) on issue 151. Seems to me like a copy/paste error when the track set change algorithm was defined.